packages/common/src/views/AccountSelect.svelte

```diff
@@ -263,18 +316,30 @@
           <h4 class="control-label">
             Select Base Path
           </h4>
-          <select
-            class='base-path-select'
-            bind:value={scanAccountOptions['derivationPath']}
-          >
-            {#each basePaths as path, pathIndex}
-              <option
-                value={path.value}
-              >
-                {path.label} - {path.value}
+          {#if customDerivationPath}
+            <input type='text' 
+              class='base-path-select' 
+              placeholder="type/your/custom/path..." 
+              on:change={handleCustomPath}
+            /> 
+            <span class='input-select' on:click={toggleDerivationPathToDropdown}></span>
+          {:else if  !customDerivationPath}
```

Comment:
Is this missing parenthesis?

Reply:
custom derivation path is a boolean, so just checking the truthy-ness of it here...is that what you are referring to?